packages/render-fidelity-tools/test/config.json

```diff
@@ -358,6 +358,166 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-ToyCar",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/ToyCar/glTF-Binary/ToyCar.glb",
+      "orbit": {
+        "radius": 0.037,
```

Comment:
lol, looks like they're toy car is smaller than a matchbox car! I kind of doubt that was intentional... Probably worth filing a bug on their repo stating its dimensions (all glTF models are in meters).

Reply:
How do I file an issue related to this?
https://screenshot.googleplex.com/3Jqj646xCh3fm4E

What information should I include? Do you happen to have a sample issue?

Reply:
No, you can just file an issue on their github: https://github.com/KhronosGroup/glTF-Sample-Models/issues

If you load the model in our editor, go to dev tools Elements section and select the model-viewer element, you can put `$0.getDimensions()` in the console to see the model's size. We really need to add this feature to our UX...